desktop/menus/view-menu.js

```diff
@@ -132,6 +105,24 @@ const buildViewMenu = settings => {
       {
         type: 'separator',
       },
+      {
+        label: '&Zoom In',
+        accelerator: 'CommandOrControl+Plus',
+        click: appCommandSender({ action: 'increaseFontSize' }),
+      },
+      {
+        label: '&Zoom Out',
```

Comment:
The `&` is for the [access key feature on Windows](https://github.com/electron/electron/issues/1698), and they should be set on unique characters for each menu level. (Sorry for the lack of documentation there!) I think we can go with `Zoom &Out` here.